apps/system/lockscreen/js/lockscreen_notification_builder.js

```diff
@@ -59,97 +59,19 @@
     }
 
     node.addEventListener('click',
-      this.requestHighlight.bind(this, node));
+      this.requestActivate.bind(this, node));
```

Comment:
I have tested it in emulator, click event is getting fired when "Ok" button is pressed.